internal/versionsapi/cli/add.go

```diff
@@ -0,0 +1,277 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/edgelesssys/constellation/v2/internal/logger"
+	"github.com/edgelesssys/constellation/v2/internal/versionsapi"
+	verclient "github.com/edgelesssys/constellation/v2/internal/versionsapi/client"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap/zapcore"
+	"golang.org/x/mod/semver"
+)
+
+func newAddCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add",
+		Short: "Add a new version",
+		Long:  "Add a new version to the versions API.",
+		RunE:  runAdd,
+	}
+
+	cmd.Flags().String("ref", "", "Ref of the version to add")
+	cmd.Flags().String("stream", "", "Stream of the version to add")
```

Comment:
I had to look up the valid options for `stream` maybe we can add them to the help text?

Reply:
Like with remove, this isn't a command meant to be executed by a dev on a daily basis, especially not without deep knowledge of the API. The allowed streams depend on the ref.